.github/actions/opensearch/action.yml

```diff
@@ -26,3 +30,4 @@ runs:
     STACK_VERSION: ${{ inputs.stack-version }}
```

Comment:
We don't use any stack, lets call it cluster version

Reply:
Done.